### ISSUE TITLE
[Feature request] Hotkeys: gamepad support and hold option

### ISSUE THREAD

**Issue by EddyHg80** [citra-emu/citra#4932]
I know these features will eventually come but I wanted to post this here as a reminder.

1) **Gamepad support**: self explanatory, maybe something like this in the gui
![Annotazione 2019-09-15 154322](https://user-images.githubusercontent.com/9092453/64922544-a309ab00-d7d0-11e9-9812-60dd32a3b7d0.png)

2) **Hold option** for:
- Speed limit toggle
- Swap screens
- Advance frame (? I don't actually know what this does sorry)

3) Reorganize in **sections**:
Maybe "general" would have load file/exit citra/capture screenshot";
"Emulation" would have restart/stop/pause emulation/speed limit; 
"Screen" would have fullscreen/toggle filter bar/toggle screen layout/toggle status bar 
and so on

I also opened this issue so I could close #3417 eheh

**Comment by legend800** [comment]
'Hold option' would be perfect for the mic as well. 

Mic still doesn't have a hotkey, forcing users to use a real mic for proper playability:
https://github.com/citra-emu/citra/issues/4720

**Comment by InfamousKnight** [comment]
For a workaround, you can use keysticks to map keys to a gamepad. Keysticks is a program for windows.

**Comment by Comitant** [comment]
Would really appreciate if this was added even though there are workarounds, just a nice feature to have.

**Comment by EddyHg80** [comment]
> Would really appreciate if this was added even though there are workarounds, just a nice feature to have.

One day german77 (yuzu dev) will port project Kraken and related to Citra and this will become reality, untill then we wait patiently.

**Comment by darrenhaken** [comment]
Has this moved along at all?

**Comment by InfamousKnight** [comment]
Not really. The fact citra is implemented in bizhawk pretty much solves the problem

**Comment by darrenhaken** [comment]
How does it solve the problem?

**Comment by InfamousKnight** [comment]
Actually, it doesn't really solve the problem for upstream citra.

Just ignore what I said
